src/pages/SignUp/SignUp.scss

```
@@ -0,0 +1,122 @@
+body {
+  background-color: #206648;
+}
+
+.signUp {
+  .container {
+    margin: 200px;
+    display: flex;
+    flex-direction: column;
+    background-color: white;
+    padding: 50px;
+  }
+
+  .nav {
+    font-size: 20px;
+    border-bottom: #e9e9e9 2px solid;
+    margin-bottom: 50px;
+    padding-bottom: 35px;
+    text-align: center;
+  }
+
+  .users {
+    margin-bottom: 40px;
+
+    .user {
+      background-color: #f8f8f8;
+      font-size: 13px;
+      border-bottom: 1px #e9e9e9 solid;
+      padding: 10px 0;
+      text-align: center;
+    }
+
+    .detail {
+      color: #606060;
+      cursor: pointer;
+    }
+  }
+  .userRulePlusOff {
+    display: none;
+  }
+
+  .userRulePlusOn {
+    display: block;
+    padding: 20px 60px;
+    text-align: left;
+    color: #797979;
+    font-size: 14px;
+
+    .title {
+      font-size: 24px;
+      text-align: center;
+      margin-bottom: 10px;
+    }
+    p {
+      margin-bottom: 10px;
+    }
+  }
+
+  .userInfoPlusOff {
+    display: none;
+  }
+
+  .userInfoPlusOn {
+    display: block;
+    padding: 20px 60px;
+    text-align: left;
+    color: #797979;
+    font-size: 14px;
+
+    .title {
+      font-size: 24px;
+      text-align: center;
+      margin-bottom: 10px;
+    }
+    p {
+      margin-bottom: 10px;
+    }
+  }
+  input {
+    margin: 10px;
+  }
+
+  .check {
+    font-size: 8px;
+  }
+
+  .useragreement {
+    margin-bottom: 35px;
+  }
+  table {
+    border: 1px solid #444444;
+    border-collapse: collapse;
+    font-size: 8px;
+    margin-bottom: 10px;
+
+    td,
+    th {
+      padding: 5px;
+      width: 400px;
+      text-align: center;
+      border: 1px solid #444444;
+    }
+  }
+
+  .nextBnt {
+    background-color: rgb(32, 102, 72);
+    width: 150px;
+    height: 50px;
+    font-size: 15px;
+    color: white;
+    border-style: none;
+    cursor: pointer;
+    position: relative;
+    left: 84%;
+    margin-top: 20px;
+  }
+
+  .nextBnt:hover {
+    background-color: #454545;
+    transition: all 0.5s ease-in-out 0.1s;
+  }
```

```suggestion
    margin-top: 20px;
    
    &:hover {
      background-color: #454545;
      transition: all 0.5s ease-in-out 0.1s;
    }
```
- 위와 같이 연산자를 활용하여 nesting 해줄 수 있습니다
- Bnt보다는 button이나 btn이 좀 더 버튼에 가까운 네이밍인거 같아요

---

수정완료 했습니다